values.yaml

```diff
@@ -59,7 +59,7 @@ injector:
   # containers.  This should be set to the official Vault image.  Vault 1.3.1+ is
   # required.
   agentImage:
-    repository: "vault"
+    repository: "hashicorp/vault"
```

Comment:
I believe `server.image.repository` also needs to be updated like this.

Reply:
Fixed.